lte/gateway/c/core/oai/test/spgw_task/test_spgw_procedures_dedicated_bearer.cpp

```diff
@@ -19,6 +19,8 @@
 
 #include "lte/gateway/c/core/oai/test/spgw_task/spgw_test_util.h"
 #include "lte/gateway/c/core/oai/test/spgw_task/spgw_procedures_test_fixture.hpp"
+#include "lte/gateway/c/core/oai/tasks/sgw/pgw_handlers.hpp"
+#include "lte/gateway/c/core/oai/tasks/sgw/pgw_procedures.hpp"
```

Comment:
Move these to after the extern "C" block, along with "spgw_test_util.h". Ref: [Ordering of headers style guide](https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes)